src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/WrapWithTag/WrapWithTagParams.cs

```diff
@@ -16,40 +18,29 @@ internal class WrapWithTagParams : ITextDocumentIdentifierParams, IRequest<WrapW
         /// Gets or sets the identifier for the text document to be operate on.
         /// </summary>
         [JsonProperty("_vs_textDocument")]
-        public TextDocumentIdentifier TextDocument
-        {
-            get;
-            set;
-        }
+        public TextDocumentIdentifier TextDocument { get; set; }
 
         /// <summary>
         /// Gets or sets the selection range to be wrapped.
         /// </summary>
         [JsonProperty("_vs_range")]
-        public Range Range
-        {
-            get;
-            set;
-        }
+        public Range? Range { get; set; }
 
         /// <summary>
         /// Gets or sets the wrapping tag name.
         /// </summary>
         [JsonProperty("_vs_tagName")]
-        public string TagName
-        {
-            get;
-            set;
-        }
+        public string? TagName { get; set; }
 
         /// <summary>
         /// Gets or sets the formatting options.
         /// </summary>
         [JsonProperty("_vs_options")]
-        public FormattingOptions Options
+        public FormattingOptions? Options { get; set; }
+
+        public WrapWithTagParams(TextDocumentIdentifier textDocument)
         {
-            get;
-            set;
+            TextDocument = textDocument;
```

Comment:
I have previously had a lot of problems defining non-empty constructors for classes which will be deserialized, but it seems it works in this instance?

Reply:
I made this change on a machine that couldn't test them, and so this didnt occur to me, but you're right this is odd. I guess whatever is deserializing here is using the "CreateUnitialized..." whatever method that serialization has. I did test everything on a real machine after the changes, but will double check.